File: 0724-find-pivot-index/0724-find-pivot-index.ts
function pivotIndex(nums: number[]): number {

    let sum: number = 0;
    for (let i = 0; i < nums.length; i++) {
        sum += nums[i];
    }
    let left: number = 0;
    for (let i = 0; i < nums.length; i++) {
        if (left == sum - left - nums[i]) {
            return i;
        }
        left += nums[i];
    }
    return -1;
};

File: 0724-find-pivot-index/0724-find-pivot-index.py
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        idx=-1
        f=list(accumulate(nums))
        b=list(accumulate(nums[::-1]))[::-1]
        for i in range(len(nums)):
            if f[i]==b[i]:
                return i
        return idx
        